schema.sql

```diff
@@ -101,3 +101,16 @@ Create Table visits(
 		Foreign Key (vet_id)
 			References vets(id)
 );
+
+---------------Performance Audit--------------------
+Begin;
+	--  Create an index on the `animal_id` column
+	CREATE INDEX idx_animal_id
+	ON visits (animal_id);
+	--  Create an index on the `vet_id` column
+	CREATE INDEX idx_vet_id
```

Comment:
- [ ] Hello! 👋 Great job on adding the index to vet_id for performance optimization. However, based on the provided screenshot, it seems that the query still opted for a sequential scan instead of utilizing the newly created index. 📈
To ensure that the index is indeed improving performance, I recommend the following steps:

1. Drop the existing index.

2. Add the index again.

3. Re-run the query with "EXPLAIN ANALYZE" to confirm that an index scan is now in use.

```sh
-- Drop the index if it exists
DROP INDEX IF EXISTS idx_vet_id;

-- Add the index
CREATE INDEX idx_vet_id ON your_table(vet_id);

-- Run the query with EXPLAIN ANALYZE
EXPLAIN ANALYZE SELECT * FROM your_table WHERE vet_id = 2;
```